tasks/framework-tools/src/lib/framework.mjs

```diff
@@ -139,9 +140,19 @@ export function packageJsonName(packageJsonPath) {
 /**
  * Build Redwood packages.
  */
-export function buildPackages(packages = frameworkPkgJsonFiles()) {
+export function buildPackages(
+  packages = frameworkPkgJsonFiles(),
+  { clean } = { clean: false }
+) {
   const packageNames = packages.map(packageJsonName)
 
+  // clean before building
+  if (clean) {
+    for (const packageJsonPath of packages) {
+      rimraf.sync(path.join(path.dirname(packageJsonPath), 'dist'))
+    }
+  }
```

Comment:
`rimraf.sync` doesn't take arrays of filepaths until v4 so we'll have to loop it for now